ui/src/components/Form/input/index.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames/bind';
+import { connect, FormikContextType, getIn } from 'formik';
+import styles from './style.module.scss';
+
+const c = classNames.bind(styles);
+
+interface TextFieldProps {
+  name: string;
+  className?: string;
+  placeholder?: string;
+  type?: string;
+}
+
+const TextField: React.FC<TextFieldProps & { formik: FormikContextType<{}> }> = ({
+  formik,
+  ...p
+}) => {
+  const error = getIn(formik.errors, p.name);
+  const inputProps = formik.getFieldProps(p.name);
+
+  const MaybeErrorMessage = () => {
+    if (!error) {
+      return null;
+    }
+
+    return <span className={c('error-icon')} title={error} />;
+  };
+
+  console.warn('INPUT PROPS', inputProps);
```

Comment:
No console. This should've given an lint error. I'll add an issue for integrating lint-staged in this repo.